dashboard/app/views/pd/application/teacher1920_application_mailer/principal_approval.html.haml

```diff
@@ -28,11 +28,11 @@
   to see the cost of the program in your region.
 
 %p
-  For participating teachers we require the following commitment:
+  We are looking for teachers who are hoping to teach the course in the 2019-20 school year.
+  In addition, we require the following commitment:
   %ul
-    %li To attend one, five-day summer workshop in 2019
+    %li To attend one, five-day summer workshop in 2019,
```

Comment:
this line and the next shouldn't have the first comma: `To attend one five-day summer workshop in 2019,`